lib/assetmanager.js

```diff
@@ -496,8 +497,8 @@ exports = module.exports = {
                       logger.log('error', 'Failed to delete stored file in assetmanager', storedFile.path);
                     }
 
-                    // return the creation error
-                    return next(createError);
+                    // Indicate that an error has occurred
+                    return res.status(500).json({success: false, message: "An error occured during upload, please contact an Administrator."});
```

Comment:
Hi @taylortom, yes it should.  We don't seem to have localised any of the Node.js error messages right now though.  Should probably revisit them all with one ticket, like we did for the front end.